code/modules/shuttle/marine_dropship.dm

```diff
@@ -607,6 +610,24 @@
 			return
 		do_hijack(M, CT, X)
 
+	if(href_list["abduct"])
+		var/groundside_humans
+		for(var/mob/H in GLOB.alive_human_list)
+			if(H.z != X.z)
+				continue
+			groundside_humans++
+
+		if(groundside_humans > 5)
+			to_chat(X, "<span class='xenowarning'>There is still prey left to hunt!</span>")
+			return
+
+		var/confirm = alert(usr, "Would you like to capture the metal bird?\n THIS WILL END THE ROUND", "Capture the ship?", "Yes", "No")
```

Comment:
```suggestion
		var/confirm = tgui_alert(usr, "Would you like to capture the metal bird?\n THIS WILL END THE ROUND", "Capture the ship?", list( "Yes", "No"))
```